feat(changelog): containers-changed-increase-maxscale-limit-to-50 2024-06-11 (#3294)
scaleway/docs-content: changelog/june2024/2024-06-11-containers-changed-increase-maxscale-limit-to-50.mdx

---

> title: Increase max_scale limit to 50

Reviewer: ```suggestion
title: Serverless Containers upgrade max scale limit to 50
```

> title: Serverless Containers upgrade max scale limit to 50

Reviewer 2: ```suggestion
title: Serverless Containers maximum scale raised to 50 instances
```

status: changed
author:

>     fullname: 'Join the #[serverless-containers] channel on Slack.'

Reviewer: ```suggestion
    fullname: 'Join the #serverless-containers channel on Slack.'
```

    url: 'https://slack.scaleway.com'
date: 2024-06-11
category: serverless
product: containers
---

> Users can now define a max_scale limit of 50 container instances, instead of 20 before.

Reviewer: ```suggestion
Serverless Containers can now scale up to 50 instances instead of 20 before for more demanding workloads. 
```

> Serverless Containers can now scale up to 50 instances instead of 20 before for more demanding workloads. 

Reviewer 2: ```suggestion
Serverless Containers can now scale up to 50 instances instead of 20, for your most demanding workloads. 
```
